label_studio/core/settings/base.py

```diff
@@ -108,6 +108,14 @@
 DEBUG = get_bool_env('DEBUG', True)
 DEBUG_MODAL_EXCEPTIONS = get_bool_env('DEBUG_MODAL_EXCEPTIONS', True)
 
+# Whether to verify SSL certs when making external requests, eg in the uploader
+# ⚠️ Turning this off means assuming risk. ⚠️
+# Overridable at organization level via Organization#verify_ssl_certs
+VERIFY_SSL_CERTS = get_bool_env('VERIFY_SSL_CERTS', True)
```

Comment:
Please let our customers know about this VERIFY_SSL_CERTS var before they roll out this version

Reply:
:100: